src/components/new-request/business-fetch.vue

```diff
@@ -59,7 +59,7 @@ export default class BusinessFetch extends Vue {
   private validate (): boolean {
     if (!this.searchField) {
       this.errorMessages = ['Required field']
-    } else if (!/^(A|BC|C|CP|FM|LLC|LP|S|XCP|XL|XP|XS)( |)\d{7}$/i.test(this.searchField)) {
+    } else if (!/^(A|BC|C|CP|FM|LLC|LP|PA|PAR|S|XCP|XL|XP|XS)( |)\d{7}$/i.test(this.searchField)) {
```

Comment:
I am just going to resolve 17952 at the same time as this ticket. This change, plus Vysakh's ticket were all that was needed.

Side note: Would it be valuable to come up with a generic solution here? For example: changing this regex to just check the generic format of having a 1-3 letter code followed by 7 digits, rather than checking the codes explicitly? Otherwise we will have to remember to update this specific section of code whenever a legal type code is added or changed.

Reply:
That's a really good idea.

It would also be good to display a useful error message if the format is invalid.

Reply:
Good call, I'll run the idea past UX

Reply:
Updated the regex, just waiting to hear back from UX on the wording of the error message.

Reply:
This all looks good to me.

When you hear back, if everything is OK, merge away.